minimalTest.js

```diff
@@ -83,7 +84,15 @@ function render() {
             return acc + `<div><strong>${senderName}: </strong> ${message.event.content.body} </div>`
         }, '')
 
-        document.getElementById("view").innerHTML = messageHistoryHTML
+      
+        view.innerHTML = messageHistoryHTML
+        
+        // Autoscroll to new message, when scrollbar is at the bottom of chatbox
+        const isScrolledToBottom = view.scrollHeight - view.clientHeight <= view.scrollTop + 1
+      
+        if (isScrolledToBottom) {
+          view.scrollTop = view.scrollHeight - view.clientHeight
+        }
```

Comment:
probably wanna move this into the `if (viewingRoom === null)` block above, and if possible do a scroll to top for `else`, so that it always scrolls to top where the most recent chatroom is if no chatroom is selected or scroll to bottom where the most recent message is if a chatroom is selected.

Alternatively we can just reverse the sort order of chatrooms such that the most recent chatroom is at the bottom and we always scroll to bottom

Reply:
But it should be there no? It's logic for the chat. Added auto scroll to top when no room is selected

Reply:
ah yeah sry i read that wrong, all good